src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -1260,6 +1293,7 @@ def sendJobAccounting(self, status="", minorStatus=""):
             "JobType": self.jobType,
             "JobClass": self.jobClass,
             "ProcessingType": self.processingType,
+            "Site": self.siteName,
```

Comment:
Is it really needed? Is it added specifically for the case of PushJobAgent?

Reply:
By default, `JobAccounting` uses `DIRAC.siteName()` as `Site` if it is not passed through this dictionary.
`DIRAC.siteName()` returns the value of `/LocalSite/Site` from the local cfg file.

The local cfg is always present when the `JobWrapper` is executed from a Pilot-Job, but not from the `PushJobAgent`.
In the latter case then, `DIRAC.siteName()` will return the name of the machine.

To avoid that, I explicitly define `Site` as `self.siteName` in the `JobAccounting` dictionary here. 
The value of `self.siteName` is `DIRAC.siteName()` by default (see in `JobWrapper.__init__`).
In `JobWrapper.initialize()` I added a line to overwrite `self.siteName` only if `Site` is present within the CE parameters.

This is the case for the `PushJobAgent` (the CE parameters are enriched with Site information with `QueueUtilities.py`) and should also be the case for the pilot-jobs I guess (`ComputingElement` uses the local cfg to get more parameters with `initializeParameters()`).